examples/compute_plan/scripts/add_compute_plan.py

```diff
@@ -24,6 +24,7 @@
 compute_plan_keys_path = os.path.join(current_directory, '../compute_plan_keys.json')
 
 client = substra.Client.from_config_file(profile_name="node-1")
+client.login(username="node-1", password="p@$swr0d44")
```

Comment:
We originally had the login call in the examples but every login call invalidate the previous token. As a result, you would have to login using the CLI after each script.

This is why we decided to include the login command in the readme as a prerequisite.

Reply:
Yes I understand, and I cheated a bit here since the SDK login is on profile 'node-1' while the CLI login is on profile 'default' so they do not invalidate each other.

I think this is not ideal, but as we discussed it in https://github.com/SubstraFoundation/substra/issues/200 it is not simple to solve.

Reply:
Actually they do, because they are for the same username. The token is invalidated in the backend, not just in the local profile.